### PR TITLE
Update documentations to new format

### PR DESCRIPTION
**Pull request by kapilraajP**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: sqs/Module.md [hunk 1]
```diff
@@ -1,83 +1,25 @@
-[![Build Status](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs)
+## Overview
+Ballerina connector for Amazon SQS is connecting the Amazon SQS API via Ballerina language easily. It provides capability to perform operations related to queues and messages.
```
Comment (RukshiW) [inline sqs/Module.md:2]: Change "is connecting" to "connects" and "easily" to 'with ease"

File: sqs/Module.md [hunk 2]
```diff
@@ -1,83 +1,25 @@
...
 
-# Ballerina Amazon SQS Connector
+This module supports [Amazon SQS API](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/welcome.html) version 2012-11-05.
 
-Amazon SQS Connector allows you to connect to the Amazon Simple Queue Service (SQS) via REST API from Ballerina.
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [AWS account](https://aws.amazon.com)
```
Comment (RukshiW) [inline sqs/Module.md:8]: Make "Create" also a part of the link text.

File: sqs/Module.md [hunk 3]
```diff
@@ -1,83 +1,25 @@
...
+* Obtaining tokens
```
Comment (RukshiW) [inline sqs/Module.md:9]: Suggestion: "Obtain tokens."

Isn't there a direct link we can use with the above as link text?

File: sqs/Module.md [hunk 4]
```diff
@@ -1,83 +1,25 @@
...
+        
+    Follow [this link](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) and obtain the Access key ID, Secret access key, Region and Account number from navigation pane `Users` and selecting `Security credentials` tab.
```
Comment (RukshiW) [inline sqs/Module.md:11]: Can we convert this to some clear steps? Check the following:

1 [Create and activate your AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
2. To obtain the access key ID, secret access key, region and account number, click **Users** on the navigation pane and then click **Security Credentials**."

Is substep 1 the same as step 1? If so, have only the second step as text under "Obtain Tokens".

File: sqs/Module.md [hunk 5]
```diff
@@ -1,83 +1,25 @@
...
-3. Run tests :
-```ballerina
-bal test
-```
-
-# Quickstart
-
-## Import the AWS SQS module
+### Step 1: Import connector
 First, import the ballerinax/aws.sqs module into the Ballerina project.
```
Comment (RukshiW) [inline sqs/Module.md:18]: Suggestion: "First, import the ballerinax/aws.sqs module into the Ballerina project as follows."

File: sqs/Module.md [hunk 6]
```diff
@@ -1,83 +1,25 @@
...
-
 ```ballerina
 import ballerinax/aws.sqs;
 ```
-
-## Initialize the AWS SQS Client giving necessary credentials
+### Step 2: Create a new connector instance
 
 You can now enter the credentials in the SQS client configuration and create SQS client by passing the configuration:
```
Comment (RukshiW) [inline sqs/Module.md:24]: Suggestion: "You can now enter the credentials in the SQS client configuration and create the SQS client by passing the configuration as follows."

File: sqs/Module.md [hunk 7]
```diff
@@ -92,300 +34,20 @@ sqs:Configuration configuration = {
...
-        accountNumber: accountNumber
-    };
-
-    sqs:Client sqsClient = check new (configuration);
+### Step 3: Invoke connector operation
 
-    // Declare common variables
-    string queueResourcePath = "";
-    string receivedReceiptHandler = "";
+1. You can create a queue in SQS as follows with `createQueue` method for a preferred queue name and the required set of attributes. Successful creation returns the created queue URL as a string and the error cases returns an `error` object.
```
Comment (RukshiW) [inline sqs/Module.md:39]: Change "error cases returns an `error` object." to "error cases return an `error` object.".

If this is going to be a substep, use the imperative tone.

File: sqs/Module.md [hunk 8]
```diff
@@ -92,300 +34,20 @@ sqs:Configuration configuration = {
...
-1. Clone this repository using the following command:
-    ```shell
-    $ git clone https://github.com/ballerina-platform/module-ballerinax-aws.sqs.git
-    ```
-
-2. Run this command from the `module-ballerinax-aws.sqs` root directory:
-    ```shell
-    $ bal build
     ```
+2. Use `bal run` command to compile and run the Ballerina program.
```
Comment (RukshiW) [inline sqs/Module.md:51]: Shouldn't this be step 4?

Comment (kapilraajP) [reply sqs/Module.md:51]: No it is under step 3, as other connector documentation

File: sqs/Module.md (hunk 6 above)
Comment (kapilraajP) [reply sqs/Module.md:24]: Fixed

File: sqs/Module.md (hunk 5 above)
Comment (kapilraajP) [reply sqs/Module.md:18]: Fixed

File: sqs/Module.md (hunk 3 above)
Comment (kapilraajP) [reply sqs/Module.md:9]: Fixed

File: sqs/Module.md (hunk 4 above)
Comment (kapilraajP) [reply sqs/Module.md:11]: Fixed by adding as one line

File: sqs/Module.md (hunk 1 above)
Comment (kapilraajP) [reply sqs/Module.md:2]: Fixed

File: README.md [hunk 9]
```diff
@@ -1,391 +1,45 @@
...
-        receivedReceiptHandler = response3[0].receiptHandle;
-    }
+1. Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
 
-    // Delete the received the message from the queue
-    error? response4 = sqsClient->deleteMessage(queueResourcePath, receivedReceiptHandler);
-    if (response4 is ()) {
-        log:printInfo("Successfully deleted the message from the queue.");
-    }
+    > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed JDK.
```
[the remark this answers is not in the document]
Comment (kapilraajP) [reply README.md:19]: Fixed

File: README.md [hunk 10]
```diff
@@ -1,391 +1,45 @@
...
-    attributes["FifoQueue"] = "true";
+This connector provides operations for connecting and interacting with Amazon SQS endpoints over the network. 
 
-    string|error response1 = sqsClient->createQueue("demo.fifo", attributes);
-    if (response1 is string) {
-        log:printInfo("Created queue URL: " + response1);
-        // Keep the queue URL for future operations
-        queueResourcePath = sqs:splitString(response1, "amazonaws.com", 1);
-    }
+For more information, go to the module(s).
```
[the remark this answers is not in the document]
Comment (kapilraajP) [reply README.md:11]: Fixed

File: sqs/Module.md (hunk 2 above)
Comment (kapilraajP) [reply sqs/Module.md:8]: That is not how it is written in other connector as well

**Review by abeykoon — APPROVED** [review]

**Pull request merged**